elastic4s-http/src/main/scala/com/sksamuel/elastic4s/http/index/IndexImplicits.scala

```diff
@@ -23,7 +23,11 @@ trait IndexImplicits extends IndexShowImplicits {
       }
 
       val params = scala.collection.mutable.Map.empty[String, String]
-      request.opType.foreach(opType => params.put("op_type", opType.lowercase))
+      request.createOnly.foreach(createOnly => 
+        if(createOnly) {
+          params.put("op_type", "create")
```

Comment:
What is the name for the field in the Elasticsearch codebase now? "createOnly" But the param on the http client is still called "op_type". :/